javascript/entities/player.js

```diff
@@ -468,6 +471,10 @@ class Player {
     pawnInvoke('GivePlayerMoney', 'ii', this.id_, amount);
   }
 
+  // Settings for the player stored inside the database.
+  get settings() {
```

Comment:
I like this! `gunther.settings.getValue('abuse/ban_on_sight')` for the win 👍

Reply:
No, don't ban gunther!
#BringBackGunther